labs/crazyegg-integration/metadata.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Crazy Egg Integration
+summary: Crazy Egg integration allows you to see heatmaps with Optimizely data. For more information on how to use this integration, please take a look at this Knowledge Base article: https://help.optimizely.com/Integrate_Optimizely_with_Crazyegg
+revisionDate: '2021-04-05'
+labels:
+  - web
+  - analytics
+  - crazyegg
+  - forms
+  - heatmap
+
+excludeFromListing: false
```

Comment:
Let's set this to `true` at first. Once the page is live and everything looks good, we'll set it to `false` so it appears in the main listing and users can find it.